deploy/template.yaml

```diff
@@ -86,8 +86,8 @@ Mappings:
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
       TXMAMESSAGERETENTIONPERIODDAYS: 604800 # Default: 7 days
       GOVUKNOTIFYTEMPLATEID: "bea2a584-4dca-4970-a90d-93977e752fdf"
-      CLIENTSESSIONTOKENTTL: 950400 # Default 11 days
-      RESOURCESTTL: 1036800 # Default 12 days
+      YOTISESSIONTTLDAYS: 10 # Default 10 days
+      RESOURCESTTL: 1209600 # Default 14 days
```

Comment:
can `RESOURCETTL`  be kept consistent pls i.e. `RESOURCETTLSECS`

Reply:
Done